sample/bin/setup.js

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env node
+
+const { main, MainError } = require('@effection/node');
+const rmrfsync = require('rimraf').sync;
+const fs = require('fs');
+const fsp = fs.promises;
+
+const { 
+  messages, TARGET_DIR, SOURCE_DIR, startScript
+} = require('./constants');
+const { formatErr, formatSuccess, spin } = require('./console-helpers');
+const { install } = require('./install');
+
+async function createDirectory(message) {
+  if (fs.existsSync(TARGET_DIR)) {
+    throw new MainError({ 
+      message: `${formatErr('directory \'bigtest-sample\' already exists')}\n${messages.abort}`
+    });
+  } else {
+    await fsp.mkdir(TARGET_DIR);
+    console.log(`\n${formatSuccess(message)}`);
+  };
+};
+
+function* migrate(messages) {
+  yield spin(messages.before, function* () {  
+    yield fsp.readdir(SOURCE_DIR).then(files => files.forEach(file => {
+      if(file === 'package.json'){
+        const {
+          name, version, description, repository, author, license, 
+          main, scripts, devDependencies, eslintConfig, browserslist,
+          babel, jest
+        } = require(`${SOURCE_DIR}/${file}`);
+  
+        scripts.start = startScript;
+  
+        const pkgjson = {
+          name, version, private: true, description, repository, author, 
+          license, main, scripts, devDependencies, eslintConfig, browserslist,
+          babel, jest
+        };
+  
+        fs.writeFileSync(`${TARGET_DIR}/package.json`, JSON.stringify(pkgjson, null, 2));
+      } else if(file !== 'bin') {
+        fs.renameSync(`${SOURCE_DIR}/${file}`, `${TARGET_DIR}/${file}`);
+      };
+    }));
+  });
+  console.log(formatSuccess(messages.after));
+};
```

Comment:
@cowboyd with the help of `path.dirname(__dirname)` as `SOURCE_DIR` the script now grabs the necessary files from the cache location of the npx install.

However, with effection in our dependencies, I've been seeing this install error. It doesn't affect the result of the install script but still:
![Screen Shot 2021-01-14 at 9 45 27 PM](https://user-images.githubusercontent.com/29791650/104675079-bffb0a80-56b2-11eb-8e97-964d5cf5326a.png)

Reply:
*sigh* Ok, looks like a mini-yak that needs shaving. Let me have a look

Reply:
Ok, looks like the problem is that we didn't test our install script on npm, just yarn for `ctrlc-windows`. I think the same error is occuring, but yarn just sits on it, while npm is noiser about it. I think I have a fix...

Reply:
https://github.com/thefrontside/ctrlc-windows/pull/19